<?xml version="1.0" encoding="utf-8"?>

<launch>
  <arg name="pointlaser_config_file"
       default="cfg/pointlaser_params.yaml"
       doc="YAML file holding the parameters."/>
  <arg name="distobox_usb_port"
       default="/dev/serial/by-id/usb-Silicon_Labs_CP2103_USB_to_UART_Bridge_Controller_0001-if00-port0"
       doc="Path of the USB port that the distobox is connected at."/>
  <arg name="move_arm_manually" default="false"/>
  <!-- Calibration node: tf transforms between lasers and reference link -->
  <include file="launch/lasers_tf_publishers_kinova.launch">
  </include>
  <!-- pointlaser_loc node -->
  <node name="pointlaser_loc"
        pkg="cpt_pointlaser_loc_ros"
        type="kinova_localizer_node"
        output="screen"
        respawn="true">
    <rosparam command="load" file="$(find cpt_pointlaser_loc_ros)/$(arg pointlaser_config_file)" />
    <param name="move_arm_manually" value="$(arg move_arm_manually)" />
  </node>
</launch>
